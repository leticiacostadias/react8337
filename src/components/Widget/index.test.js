import React from 'react';
import Widget from './index';
import renderer from 'react-test-renderer';

test('Teste do componente Widget', () => {
  const widget = renderer.create(
    <Widget>
      <h1>TÃ­tulo do widget</h1>
    </Widget>
  );
  let arvoreDOM = widget.toJSON();

  expect(arvoreDOM).toMatchSnapshot();
});