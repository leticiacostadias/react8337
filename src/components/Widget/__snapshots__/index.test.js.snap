// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Teste do componente Widget 1`] = `
<div
  className="widget"
>
  <h1>
    TÃ­tulo do widget
  </h1>
</div>
`;
